### PR TITLE
FR new overthebox migration offer

### PR DESCRIPTION
**Pull request by AxelDum**
We now have migration offer in manager to :
* Migrate from IT/Plus to new offer Starter/Business
* Order a new Hardware OTBv3
* Migrate from Starter to Business and vice et versa

### CODE REVIEW

**Comment by AxelDum** [comment]
Missing meta.yaml don't know if I should add it

File: pages/assets/screens/control_panel/product-selection/telecom/tpl-telecom-01-fr-internet.png
Comment (AxelDum) [inline pages/assets/screens/control_panel/product-selection/telecom/tpl-telecom-01-fr-internet.png:1]: Updated to be accurate with current manager version and avoid a blurry mess

File: pages/assets/screens/control_panel/product-selection/telecom/tpl-telecom-05-fr-otb.png
Comment (AxelDum) [inline pages/assets/screens/control_panel/product-selection/telecom/tpl-telecom-05-fr-otb.png:1]: Updated to be accurate with current manager version and avoid a blurry mess

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md
Comment (AxelDum) [inline pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md:1]: Similar to internet-access change offer guide

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Objectif
+
+Vous pouvez faire évoluer votre offre OverTheBox afin de pouvoir accéder à de nouvelles fonctionnalités et services.
+
+Le changement d'offre permet de :
+
+* Migrer d'une offre Starter 500Mbps à une offre Business 1Gbps.
+* Migrer d'une offre Business 1Gbps à une offre Starter 500 Mbps.
+* Migrer d'une ancienne offre de service à une nouvelle offre Starter ou Business.
+* Commander un boitier OTBv3
```
Comment (AxelDum) [inline pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md:15]: Boitier as of 1990 orthography rules should we revert to old orthography ?
https://www.dictionnaire-academie.fr/article/A9B1480

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Une fois l'adresse saisie, cliquez sur le bouton `Sélectionner le contact`{.action}
+
+> [!primary]
+> Les frais de livraison sont inclus dans le prix du boitier
+
+![selection d'adresse de livraison](images/manager-selectAddress-2024.png){.thumbnail}
+
+### Valider le changement d'offre
+
+La page `Résumé du changement d'offre` résume la tarification mensuelle de la nouvelle offre, ainsi que le cout du boitier (si l'option a été sélectionnée).
```
Comment (AxelDum) [inline pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md:67]: Boitier and cout as of 1990 orthography rules should we revert to old orthography ?
https://www.dictionnaire-academie.fr/article/A9B1480
https://www.dictionnaire-academie.fr/article/A9C4688

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+> [!primary]
+> Pour plus d'informations sur le paiement d'un bon de commande, reportez-vous sur le guide [Gérer mes commandes OVHcloud](https://help.ovhcloud.com/csm/fr-billing-managing-ovh-orders?id=kb_article_view&sysparm_article=KB0042874)
+
+![paiement](images/manager-payment-2024.png){.thumbnail}
+
+## Aller plus loin
+
+N'hésitez pas à échanger avec notre communauté d'utilisateurs sur vos produits Télécom sur notre site [OVHcloud Community](https://community.ovh.com/c/telecom)
+
+Plus d'informations sur [les offres OverTheBox](https://www.ovhtelecom.fr/overthebox/)
```
Comment (AxelDum) [inline pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md]: Migration is available but some issue on website side for new offers pages, will update once it prod

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+### Payer le bon de commande
+
+> [!primary]
+> Une fois le bon de commande payé le changement d'offre est immédiat. Un remboursement au prorata de la facturation du mois en cours est établi.
+
+Pour finaliser le changement d'offre, ainsi que la livraison du boitier, vous devez payer le bon de commande généré lors de la confirmation de la migration.
+Pour ce faire cliquez sur le lien contenu dans la phrase `Pour finaliser la migration de votre offre, vous devez payer le bon de commande.`{.action}.
+
+> [!primary]
+> Pour plus d'informations sur le paiement d'un bon de commande, reportez-vous sur le guide [Gérer mes commandes OVHcloud](https://help.ovhcloud.com/csm/fr-billing-managing-ovh-orders?id=kb_article_view&sysparm_article=KB0042874)
```
Comment (AxelDum) [inline pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md]: Should it be a relative link?

**Comment by AxelDum** [comment]
Small update on migration delay as it is not immediate if an hardware was ordered
I'm good on my side

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md (hunk 2 above)
Comment (OVH-Team-Guides) [reply pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md:67]: Both are OK. Let's stick with your version.

File: pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md (hunk 4 above)
Comment (OVH-Team-Guides) [reply pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md]: Yes, it should be relative. I fixed it.

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by AxelDum** [comment]
ok pour moi

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @AxelDum

**Pull request merged**